origin-dapp/src/components/conversation.js

```diff
@@ -25,14 +26,44 @@ class Conversation extends Component {
       newMessagePlaceholder: {
         id: 'conversation.newMessagePlaceholder',
         defaultMessage: 'Type something...'
-      }
+      },
+      offerCreated: {
+        id: 'conversation.offerCreated',
+        defaultMessage: 'made an offer on'
+      },
+      offerWithdrawn: {
+        id: 'conversation.offerWithdrawn',
+        defaultMessage: 'withdrew their offer on'
```

Comment:
@Natasha08 these should be complete sentences, if possible. I know that we don't always do this elsewhere in the code, but we should ([like this](https://github.com/OriginProtocol/origin/commit/9ed6ead33672b2ea6c46b2b73e89926dcb54cf80#diff-61c348e20f66a533d512610bffbebad3R83)).